instructor/function_calls.py

```diff
@@ -135,6 +130,9 @@ def from_response(
         if mode == Mode.GEMINI_JSON:
             return cls.parse_gemini_json(completion, validation_context, strict)
 
+        if mode == Mode.GEMINI_TOOLS:
```

Comment:
This does functionally the same thing as the vertex_ai_tools function above. Why not we just reuse the vertexai_tools mode

Reply:
```
if mode in {Mode.VERTEXAI_TOOLS, Mode.GEMINI_TOOLS}:
            return cls.parse_vertexai_tools(completion, validation_context, strict)
```

might be good to just rename that to parse_gemini_tools too